troposphere/static/js/components/Master.jsx

```diff
@@ -55,6 +57,15 @@ export default React.createClass({
         // The code below is only relevant to logged in users
         if (!context.hasLoggedInUser()) return;
 
+        if (Raven && window.SENTRY_DSN){
```

Comment:
We are putting window vars into `globals`, this has the positive effect that we can statically guarantee via `import { SENTRY_DSN } from globals`. Otherwise window.field is more susceptible to typos etc, and its globally/mutable. Not a big deal.